docs/contribute/contribution_guidelines.rst

```diff
@@ -70,6 +70,17 @@ The testing for functions and classes should at least contain tests for
 
 In case of deprecations, it must also be tested if the deprecated feature is properly removed (see `tests/test_deprecations.py` for examples).
 
+Testing plot functions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The above also holds for plot functions, however, in this case it is required that the tests generate baseline and test images. The following documents the current best practice. See `test_plot.py <https://github.com/pyfar/pyfar/blob/main/tests/test_plot.py>`_ for a code example.
```

Comment:
I don't think that linking to the file is good practice here. People who are new and are looking for the correct information have to search the right line of code in a large file. 
Please provide a code snippet with a minimal working example